templates/oam/workloads/v1.4.1/virtualservice.mesh.meshery.layer5io.schema.json

```diff
@@ -85,5 +85,6 @@
     }
   },
   "type": "object",
+  "title":"virtualservice",
```

Comment:
Is this been dynamically generated or manually generated?

Reply:
If manual, can we set proper grammar here with capitalization and whitespace?

If manual, how are we guaranteeing new components are autogenerated properly and completely during runtime?

Reply:
In this case these are generated manually, but in the workflows as well as runtime, these look like this:
"WasmPlugin"
"DestinationRule"
"EnvoyFilter"
"Gateway"
"ServiceEntry"
"Sidecar"
"VirtualService"
"WorkloadEntry"
"WorkloadGroup"
"AuthorizationPolicy"
"PeerAuthentication"
"RequestAuthentication"
"Telemetry"

Reply:
So I should change this to VirtualService and so on and so on

Reply:
Actually, ideally "Virtual Service", if this is the value that we're using as a display name.